### PR TITLE
Enable Selinux Enforcing

### PR DESCRIPTION
**Pull request by zimmerry**

### CODE REVIEW

File: resources/git_credentials.rb [hunk 1]
```diff
@@ -8,6 +8,8 @@
 property :secrets_item,    String, default: lazy { node['osl-git']['secrets_item'] }
 property :use_http_path, [true, false], default: true
 
+unified_mode true
+
```
Comment (ramereth) [inline resources/git_credentials.rb:12]: Can you please make a separate PR which addresses this first as a Chef 17 update?

**Comment by ramereth** [comment]
@zimmerry can you please rebase this? Also remove your commit which reverts the change. That shouldn't be included anymore now that #20 is merged.

**Comment by zimmerry** [comment]
@ramereth I'm having some issues getting rid of `c2ae3c8`, this branch's history is all messed up. I'm just going to close this PR and start one fresh.
